### PR TITLE
Add tag field to OptimizerParamsHandler

### LINKED ISSUE #495: Add tag to OptimizerParamsHandler
Idea is to improve `OptimizerParamsHandler` (`tensorboard_logger`, `visdom_logger`, ...) by adding a tag free field in order to be able to log several optimizers in different plots. This can be helpful when logging lr of generator/descriminator optimizers in GAN.

```python
tb_logger.attach(trainer,
                 log_handler=OptimizerParamsHandler(optimizer_D, "lr", tag="descriminator"), 
                 event_name=Events.EPOCH_STARTED)

tb_logger.attach(trainer,
                 log_handler=OptimizerParamsHandler(optimizer_G, "lr", tag="generator"), 
                 event_name=Events.EPOCH_STARTED)
```

### PR DESCRIPTION
**Pull request by Mxbonn**
Fixes #495

Description:
Adds tag field while maintaining current behavior when no tag is given.
This is my first pull request so if I missed something let me know ;)

### CODE REVIEW

File: ignite/contrib/handlers/tensorboard_logger.py [hunk 1]
```diff
@@ -87,17 +87,22 @@ class OptimizerParamsHandler(BaseOptimizerParamsHandler):
...
+    def __init__(self, optimizer, param_name="lr", tag=None):
+        super(OptimizerParamsHandler, self).__init__(optimizer, param_name, tag)
 
     def __call__(self, engine, logger, event_name):
         if not isinstance(logger, TensorboardLogger):
             raise RuntimeError("Handler 'OptimizerParamsHandler' works only with TensorboardLogger")
 
         global_step = engine.state.get_event_attrib_value(event_name)
-        params = {"{}/group_{}".format(self.param_name, i): float(param_group[self.param_name])
+        if self.tag:
```
Comment (vfdev-5) [inline ignite/contrib/handlers/tensorboard_logger.py]: I would code it in one line : `tag_prefix = "{}/".format(self.tag) if self.tag else ""`

**Comment by vfdev-5** [comment]
@Mxbonn thanks for the PR. Looks good! I left a comment in the code. And maybe a little test can be added too. 
And I need to repair CI before merging the PR

**Comment by Mxbonn** [comment]
@vfdev-5 thanks for the comment, I addressed it and also added tests :)

**Comment by vfdev-5** [comment]
Thanks Maxim ! Looks OK for me. As previously said we'll need to fix the CI at first and then we can merge your PR.

**Pull request merged**